src/lib/es5.d.ts

```diff
@@ -1682,9 +1697,16 @@ interface ArrayBufferTypes {
 }
 type ArrayBufferLike = ArrayBufferTypes[keyof ArrayBufferTypes];
 
+/**
+ * ArrayBuffer constructor options
+ */
+interface ArrayBufferOptions {
+    maxByteLength?: number;
+}
+
 interface ArrayBufferConstructor {
     readonly prototype: ArrayBuffer;
-    new(byteLength: number): ArrayBuffer;
+    new(byteLength: number, options?: ArrayBufferOptions): ArrayBuffer;
     isView(arg: any): arg is ArrayBufferView;

```

Comment:
Hi @dsherret, I am not sure what this line **`isView(...`** should be returning but as per MDN docs, it should be returning boolean. You can check [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer/isView#return_value).

Reply:
@indrajitbnikam This *is* a boolean return type; `arg is ArrayBufferView` is syntax for a type predicate.  See https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates.

Reply:
Understood